src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -542,7 +555,7 @@ private async Task InitPeer()
             _api.DisposeStack.Push(new NodeSourceToDiscV4Feeder(enrDiscovery, _api.DiscoveryApp, 50));
         }
 
-        CompositeNodeSource nodeSources = new(_api.StaticNodesManager, nodesLoader, enrDiscovery, _api.DiscoveryApp);
+        CompositeNodeSource nodeSources = _api.DiscoveryV5App is null ? new(_api.StaticNodesManager, nodesLoader, enrDiscovery, _api.DiscoveryApp) : new(_api.StaticNodesManager, nodesLoader, enrDiscovery, _api.DiscoveryApp, _api.DiscoveryV5App);
```

Comment:
Don't see `_api.DiscoveryV5App` being assigned anywhere

Reply:
A line in the comment above

Reply:
Could you give a line link?
I only see assignments to `_api.DiscoveryApp`, nothing for `_api.DiscoveryV5App`